aiohttp/test_utils.py

```diff
@@ -57,12 +58,17 @@
     SSLContext = None
 
 
+REUSE_ADDRESS = os.name == 'posix' and sys.platform != 'cygwin'
+
+
 def get_unused_port_socket(host: str) -> socket.socket:
     return get_port_socket(host, 0)
 
 
 def get_port_socket(host: str, port: int) -> socket.socket:
     s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    if REUSE_ADDRESS:
```

Comment:
It will be nice if you add a comment describing why it is set here.

Reply:
+1

not everyone knows that it has different semantics under Windows.

Reply:
Well, I mean another thing ;). I don't understand why it ever needs. Even in Linux.

Reply:
Because if you retry binding to the same socket fast enough, it'll fail. Even from the same process (after unbinding). There's some sort of timeout in place.

OTOH if you were to always use an ephemeral port 0, the kernel would allocate you a different port each time and there wouldn't be such a problem.

Reply:
Sorry, I confused. I misread as SO_REUSEPORT.